integration_test/signup_test.go

```diff
@@ -66,6 +66,7 @@ func TestSignup(t *testing.T) {
 	assert.NotEmpty(session.RefreshToken)
 	assert.Equal("bearer", session.TokenType)
 	assert.Equal(3600, session.ExpiresIn)
+	assert.NotEmpty(session.ExpiresAt)
```

Comment:
Can you think of a way to test the actual value in `ExpiresAt`?

E.g. perhaps we could use `assert.InDelta` with the current time and our known value for `ExpiresIn`?

I think that could be a better test for this field than simply checking that it's not empty - what do you think?